lasagne/layers/helper.py

```diff
@@ -312,9 +311,9 @@ def get_output_shape(layer_or_layers, input_shapes=None):
         return all_shapes[layer_or_layers]
 
 
-def get_all_params(layer):
+def get_all_params(layer, **tags):
     """
-    This function gathers all learnable parameters of all layers below one or
+    This function gathers all parameters of all layers below one or
     more given :class:`Layer` instances, including the layer(s) itself. Its
     main use is to collect all parameters of a network just given the output
     layer(s).
```

Comment:
I'd suggest to add the additional explanation on filtering from `Layer.get_params()` here as well, because `get_all_params()` is probably what people will learn about first.